stdlib/3/builtins.pyi

```diff
@@ -221,15 +221,15 @@ class str(Sequence[str]):
     def __init__(self, o: bytes, encoding: str = ..., errors: str = 'strict') -> None: ...
     def capitalize(self) -> str: ...
     def center(self, width: int, fillchar: str = ' ') -> str: ...
-    def count(self, x: str) -> int: ...
+    def count(self, x: str, start: int = None, stop: int = None) -> int: ...  # TODO: __index__
```

Comment:
`Optional[int] = ...`. In order to make future use of SupportsIndex or similar easier, you could for now define a type alias `_Index = int`, then declare these parameters as `Optional[_Index]`. If we gain support for SupportsIndex in the future, we can then easily change the alias.